src/pages/Main/MainLecture.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function MainVideo({ lecture }) {
+  const isChangeStar = num => {
+    switch (num) {
+      case 0:
+        return '☆';
+      case 1:
+        return '★';
+      case 2:
+        return '★★';
+      case 3:
+        return '★★★';
+      case 4:
+        return '★★★★';
+      case 5:
+        return '★★★★★';
+      default:
+        return '★★★★★';
+    }
+  };
```

Comment:
```suggestion
  const isChangeStar = num => {
    switch (num) {
      case 0:
        return '☆';
      case 1:
      case 2:
      case 3:
      case 4:
      case 5:
        return '★'.repeat(num);
      default:
        return throw Error('out of range')
    }
  };
```
특별히 switch문을 쓰고 싶으시다면 이렇게 쓰시면 될거 같습니다